runtime/Java/src/org/antlr/v4/runtime/BufferedTokenStream.java

```diff
@@ -441,15 +441,17 @@ protected int previousTokenOnChannel(int i, int channel) {
 
 	@Override
 	public String getText() {
+		fill();
 		return getText(Interval.of(0,size()-1));
 	}
 
 	@Override
 	public String getText(Interval interval) {
+		lazyInit();
```

Comment:
📝 The old behavior was correct here. `getText` has a [documented](https://github.com/antlr/antlr4/blob/9e64dfc6e99f7dcf7cae492927d37290daf57734/runtime/Java/src/org/antlr/v4/runtime/TokenStream.java#L54-L56) [precondition](https://github.com/antlr/antlr4/blob/9e64dfc6e99f7dcf7cae492927d37290daf57734/runtime/Java/src/org/antlr/v4/runtime/TokenStream.java#L30-L33) ([link](https://github.com/antlr/antlr4/blob/9e64dfc6e99f7dcf7cae492927d37290daf57734/runtime/Java/src/org/antlr/v4/runtime/IntStream.java#L115-L117)) that an [initializing method](https://github.com/antlr/antlr4/blob/9e64dfc6e99f7dcf7cae492927d37290daf57734/runtime/Java/src/org/antlr/v4/runtime/IntStream.java#L15-L23) must be called before `getText` can be called.

Reply:
Agreed, however I noticed that getTokens which is logically doing a similar action, does call lazyInit() so should not getText() and getTokens() be consistent?